content/api/vision/coordinates.md

```diff
@@ -47,9 +47,33 @@ robot, exactly 1 metre away.
 
 {{% added_in update="2018-jan" feature="spherical coordinates" %}}
 
-## Diagram
+### Diagram
 
 The following diagram shows the orientation of the Cartesian axes relative to
 the camera as well as the angles which describe the spherical coordinate space.
 
 ![A diagram showing the coordinate spaces](/img/api/coordinate-spaces.svg)
+
+## Orientation Coordinates
+
+Orientation represents the rotation of a marker around the x, y, and z axes.
+These can be accessed as follows:
+
+- `rot_x_radians` / `rot_x_degrees` - the angle of rotation in radians/degrees
+   counter-clockwise about the Cartesian x axis.
+- `rot_y_radians` / `rot_y_degrees` - the angle of rotation in radians/degrees
+   counter-clockwise about the Cartesian y axis.
+- `rot_z_radians` / `rot_z_degrees` - the angle of rotation in radians/degrees
+   counter-clockwise about the Cartesian z axis. (tip: the z axis typically
+   faces the camera, and thus will seem like a clockwise rotation)
+
+Rotations are applied in order of z, y, x.
+
+The following table visually explains what positive and negative
+rotations represent.
+|  |  |
+|:-:|:-:|
+| <img src="/img/api/coordinate-orientations/m45x0y0z.png?width=200px" style="margin:0"/> `rot_x_degrees = 45` | <img src="/img/api/coordinate-orientations/m-45x0y0z.png?width=200px" style="margin:0"/>  `rot_x_degrees = -45` |
```

Comment:
Why are we using image tags here rather than the markdown format? I can't see that forcing the margin to 0 is really required here?

Reply:
the default margin makes them very spaced out

Reply:
Hmm, you're right, that's really annoying. I'll try and fix it with CSS some time!